mlir/test/Dialect/LLVMIR/invalid.mlir

```diff
@@ -1703,3 +1703,47 @@ llvm.func @wrong_number_of_bundle_tags() {
   } : (i32, i32) -> ()
   llvm.return
 }
+
+// -----
+
+llvm.mlir.global external @x(42 : i32) {addr_space = 0 : i32} : i32
+
+// expected-error@+1 {{expects type to be a valid element type for an LLVM global alias}}
+llvm.mlir.alias external @y {addr_space = 0 : i32} : !llvm.label {
+  %0 = llvm.mlir.addressof @x : !llvm.ptr
+  llvm.return %0 : !llvm.ptr
+}
+
+// -----
+
+// expected-error@+1 {{linkage must be private, internal, linkonce, weak}}
+llvm.mlir.alias appending @y2 {addr_space = 0 : i32} : i32 {
+  %0 = llvm.mlir.addressof @x : !llvm.ptr
```

Comment:
this refers to an unknown symbol, which might cause issues when the verifier order would change